From c7108e54c517660322528efa1486aed817cf6019 Mon Sep 17 00:00:00 2001
From: David Wu <david.wu@rock-chips.com>
Date: Fri, 20 Mar 2020 09:34:56 +0800
Subject: [PATCH] ARM: dts: rv1126: Use SRST_GMAC_A to reset GMAC

The correct reset of the GMAC controller is to use SRST_GMAC_A.

Change-Id: I37a2289227b1a7efebfd004885826487a3bac3c3
Signed-off-by: David Wu <david.wu@rock-chips.com>
---
 arch/arm/boot/dts/rv1126.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rv1126.dtsi b/arch/arm/boot/dts/rv1126.dtsi
index f7284895ceed..d700d52420ab 100644
--- a/arch/arm/boot/dts/rv1126.dtsi
+++ b/arch/arm/boot/dts/rv1126.dtsi
@@ -1285,7 +1285,7 @@
 			      "mac_clk_tx", "clk_mac_refout",
 			      "aclk_mac", "pclk_mac",
 			      "clk_mac_speed", "ptp_ref";
-		resets = <&cru SRST_PDGMAC_NIU_A>;
+		resets = <&cru SRST_GMAC_A>;
 		reset-names = "stmmaceth";
 
 		snps,mixed-burst;
-- 
2.35.3

